prism.js

```diff
@@ -934,22 +934,6 @@ Prism.languages.js = Prism.languages.javascript;
 
 			xhr.send(null);
 		});
-
-		if (Prism.plugins.toolbar) {
-			Prism.plugins.toolbar.registerButton('download-file', function (env) {
-				var pre = env.element.parentNode;
-				if (!pre || !/pre/i.test(pre.nodeName) || !pre.hasAttribute('data-src') || !pre.hasAttribute('data-download-link')) {
-					return;
-				}
-				var src = pre.getAttribute('data-src');
-				var a = document.createElement('a');
-				a.textContent = pre.getAttribute('data-download-link-label') || 'Download';
-				a.setAttribute('download', '');
-				a.href = src;
-				return a;
-			});
-		}
```

Comment:
I think we need to add this to the build so the website has it again.

Reply:
Since 1) the toolbar plugin isn't loaded and 2) none of our code blocks has the `data-download-link` attribute, the plugin isn't active, so why include it again?

Reply:
I assumed the toolbar plugin was loaded, my b.